src/audio/drc/drc.c

```diff
@@ -6,7 +6,6 @@
 
 #include <sof/audio/component.h>
 #include <sof/audio/data_blob.h>
-#include <sof/audio/buffer.h>
```

Comment:
why are you removing it from here? Are you not using types from in in *this* file? Recursive header inclusion isn't enough, headers should be explicitly included where used. Including in both the .c and the .h is ok

Reply:
@lyakh isn't a bit too much to include both in foo.h and foo.c ? we are not requiring this systematically.